File: sdkwork-app-sdk-go/types/cash_account_info_vo.go
package types

// 现金账户信息响应
type CashAccountInfoVO struct {
	CreatedAt string `json:"createdAt"`
	UpdatedAt string `json:"updatedAt"`
	AccountId string `json:"accountId"`
	AccountType string `json:"accountType"`
	AccountTypeName string `json:"accountTypeName"`
	OwnerId string `json:"ownerId"`
	UserId string `json:"userId"`
	AvailableBalance float64 `json:"availableBalance"`
	FrozenBalance float64 `json:"frozenBalance"`
	TotalBalance float64 `json:"totalBalance"`
	Status string `json:"status"`
	StatusName string `json:"statusName"`
	PendingBalance float64 `json:"pendingBalance"`
	TotalRecharged float64 `json:"totalRecharged"`
	TotalSpent float64 `json:"totalSpent"`
	TotalWithdrawn float64 `json:"totalWithdrawn"`
}

File: sdkwork-app-sdk-python/sdkwork_app_sdk/models/cash_account_info_vo.py
from dataclasses import dataclass
from typing import Optional, List, Dict, Any

@dataclass
class CashAccountInfoVO:
    """现金账户信息响应"""
    created_at: str = None
    updated_at: str = None
    account_id: str = None
    account_type: str = None
    account_type_name: str = None
    owner_id: str = None
    user_id: str = None
    available_balance: float = None
    frozen_balance: float = None
    total_balance: float = None
    status: str = None
    status_name: str = None
    pending_balance: float = None
    total_recharged: float = None
    total_spent: float = None
    total_withdrawn: float = None
